opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableEditionOverview.jsx

```diff
@@ -403,7 +403,7 @@ const StixCyberObservableEditionOverviewComponent = ({
                           <Field
                             component={TextField}
                             variant="standard"
-                            name="hashes.SHA-512"
+                            name="hashes_SHA-512"
                             label={t('hash_sha-512')}
```

Comment:
missing translation

Reply:
because I noticed the file attributes are never translated : 
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/39fba663-85b5-4227-b566-55215a5522ff)